core/src/com/unciv/logic/civilization/QuestManager.kt

```diff
@@ -500,7 +500,7 @@ class QuestManager {
     fun cityStateConquered(cityState: CivilizationInfo, attacker: CivilizationInfo) {
         val matchingQuests = assignedQuests.asSequence()
             .filter { it.questName == QuestName.ConquerCityState.value }
-            .filter { it.data1 == cityState.civName && it.assignee == attacker.civName}
+            .filter { it.data1 == cityState.civName && it.assignee == attacker.civName }
```

Comment:
Unrelated to this PR and without knowing what it's for (perhaps just because of that), but data1 really sound like a suboptimal variable name

Reply:
Wait until you see its counterpart, `data2` (yes, this exists) :upside_down_face: 
If I read the code correctly, the variables are used to store data for quests that differs for each quest. E.g., in the quest for finding a civ, which civ should be found, or which natural wonder should be found for the 'find natural wonder' quest, etc. While it is possible to make variables for all these, it is easier (though costing readability) to use a multi-functional variable, especially as you then don't have variables that are only used for a single quest and unused outside of that.

Reply:
In other words, a plainer way to do what subclassing is for?